castle/cms/interfaces/controlpanel.py

```diff
@@ -291,6 +291,32 @@ class ISiteConfiguration(Interface):
     )
 
 
+class ICDNSettings(Interface):
+    cdn_alternate_url_prefix = schema.TextLine(
```

Comment:
It would be useful to make sure this value is validated as a good URL, according to what is expected by the processing method.

IE: needs scheme, needs "://" and needs a domain